pom.xml

```diff
@@ -262,7 +262,7 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-javadoc-plugin</artifactId>
-				<version>3.1.1</version>
+				<version>3.2.0</version>
```

Comment:
Looks good, this progression of plugin versioning will be confirmed on release. For now, it can be merged into the staging branch.